Update functions-create-github-webhook-triggered-function.md

ericbrunner opened the pull request:
I get stuck because after forking that sample project the Issues tab was not shown.

azurecla commented:
Hi __@abaranch__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (abaranch). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
Merged by request.

Commit:
Update functions-create-github-webhook-triggered-function.md

I get stuck because after forking that sample project the Issues tab was not shown.
